examples/TargetingConsoleApp/TargetingConsoleApp.csproj

```diff
@@ -4,7 +4,6 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
```

Comment:
Is there any reason that we keep the <Nullable>enables</Nullable> in the RazorPages example?

Reply:
Because that project was made with nullable in mind. It uses it correctly. The other projects don't use nullable in the code, and it seems it was added into csproj after the fact causing compiler warnings.